### PR TITLE
Fixed CameraHelper reference in RayShooting example

### PR DESCRIPTION
**Pull request by Birdasaur**
Replaced CameraHelper with origin cube approach for deciding origin of ray. Beautified code.

Added requestFocus() to ShapeBaseSample to improve interaction 

Fixed missing bracket in css. Beautified css.

### CODE REVIEW

File: FXyz-Samples/src/main/java/org/fxyz3d/samples/utilities/RayShooting.java [hunk 1]
```diff
@@ -74,33 +77,32 @@ public class RayShooting extends ShapeBaseSample {
...
-        
-        subScene.setOnKeyReleased(ke -> {
-            // release flag
-            if (ke.getCode().equals(KeyCode.CONTROL)) {
-                fireRay = true;
-            }
+        camera.setTranslateZ(-2000);
+
+        subScene.addEventHandler(KeyEvent.KEY_PRESSED, event -> {
+            if(event.isControlDown()) {
```
Comment (jperedadnr) [inline FXyz-Samples/src/main/java/org/fxyz3d/samples/utilities/RayShooting.java:88]: Why do you need Ctrl here? Typically you could press WASD, Space, Z without the need of any modifier.
Ctrl+{WASD/Space/Z} might have system implications in different platforms.

File: FXyz-Samples/src/main/java/org/fxyz3d/samples/utilities/RayShooting.java [hunk 2]
```diff
@@ -74,33 +77,32 @@ public class RayShooting extends ShapeBaseSample {
...
+            }            
         });
 
-        subScene.setOnMousePressed(e -> {
-            if (fireRay) {
-                Point3D o = CameraHelper.pickProjectPlane(camera, e.getX(), e.getY());
+        subScene.addEventHandler(MouseEvent.MOUSE_PRESSED, e -> {
+            if (e.isControlDown()) {
+                Point3D o = new Point3D(origin.getTranslateX(), origin.getTranslateY(), 
+                    origin.getTranslateZ()); //so it looks like its a bit in front
```
Comment (jperedadnr) [inline FXyz-Samples/src/main/java/org/fxyz3d/samples/utilities/RayShooting.java:104]: s/its/it's

File: FXyz-Samples/src/main/java/org/fxyz3d/samples/utilities/RayShooting.java (hunk 1 above)
Comment (Birdasaur) [reply FXyz-Samples/src/main/java/org/fxyz3d/samples/utilities/RayShooting.java:88]: I wanted to allow the user to translate the origin cube (origin of the ray) around in space but the ShapeBaseExample already had wasd etc mapped to translating the camera.

File: FXyz-Samples/src/main/java/org/fxyz3d/samples/utilities/RayShooting.java (hunk 2 above)
Comment (Birdasaur) [reply FXyz-Samples/src/main/java/org/fxyz3d/samples/utilities/RayShooting.java:104]: oops that comment is no longer applicable. I will remove it.

File: FXyz-Samples/src/main/java/org/fxyz3d/samples/utilities/RayShooting.java (hunk 1 above)
Comment (jperedadnr) [reply FXyz-Samples/src/main/java/org/fxyz3d/samples/utilities/RayShooting.java:88]: Okay, but as I mentioned, Ctrl+{...} has some undesired effects on Mac for instance.

Comment (Birdasaur) [reply FXyz-Samples/src/main/java/org/fxyz3d/samples/utilities/RayShooting.java:88]: oh. crud. Ok I'll switch to other key codes without control. one minute...

Comment (Birdasaur) [reply FXyz-Samples/src/main/java/org/fxyz3d/samples/utilities/RayShooting.java]: see latest.

**Review by jperedadnr — APPROVED** [review]

**Pull request merged**